### ISSUE TITLE
The Ball Bases after the Construction Table creation event in survival mode

### ISSUE THREAD

**Issue by Aviv871** [yolp900/ItsJustaCharm#3]
After creating the Construction Table in survival mode the number of Ball Bases you hold decreases and then increase back after a sec, so over all stays the same as it was before the creation.
Maybe problem in the network handling?

**Comment by yolp900** [comment]
yep, I was removing the ballbase only on the client side so it didn't go to the server... Now I wonder how I didn't notice that before... Thanks!
